pyproject.toml

```diff
@@ -59,7 +59,7 @@ exclude_lines = [
   "import",
   "pragma: no cover",
 ]
-fail_under = 97
+fail_under = 80
```

Comment:
We shouldn't be lowering the coverage percentage. Some of the methods added like `_resolve_compression_requests` and `_format_output_directory` can easily have corresponding tests.